javascript/ql/src/semmle/javascript/dataflow/internal/StepSummary.qll

```diff
@@ -40,9 +40,8 @@ newtype TStepSummary =
   ReturnStep() or
   StoreStep(PropertyName prop) or
   LoadStep(PropertyName prop) or
+  CopyStep(PropertyName prop) or
```

Comment:
I am fine with this name, but I think @asgerf should weigh in too.

Reply:
It's called load-store steps elsewhere so that would be my preferred name.

Reply:
I would also have preferred to stick with `LoadStoreStep`.   
But I can't have 2 steps with the same name, and `LoadStoreStep` is already used for the binary variant where the loaded property can be different from the stored property.

Reply:
Ah, sorry I just saw this snippet out of context. LGTM then.